cameraserver/multiCameraServer/src/main/native/cpp/main.cpp

```diff
@@ -182,7 +183,8 @@ int main(int argc, char* argv[]) {
     ntinst.StartServer();
   } else {
     fmt::print("Setting up NetworkTables client for team {}\n", team);
-    ntinst.StartClientTeam(team);
+    ntinst.StartClient4();
```

Comment:
Separating the startclient/setserver seems inconvenient.

Reply:
The problem with keeping them combined is now you have to choose whether you're creating a NT3 client or a NT4 client, so if we keep the old Start functions we have to duplicate all of them.